Hush/Assets/Scripts/Player/PlayerInputManager.cs

```diff
@@ -45,26 +48,31 @@ public void OnLook(CallbackContext context)
 	public void OnLightAttack(CallbackContext context)
 	{
 		lightAttack = context.ReadValueAsButton();
+		isInCombat = context.ReadValueAsButton();
```

Comment:
porque?

Reply:
my logic was that when you're using spells or weapons you're necessarily in combat. as discussed, this can be changed depending on enemy proximity to player (aggro) though!

Reply:
probably better to have a centralized object "GameState" that's not in the player so that we can refer to and set these kinds of variables "statically". That way you would have a variable "IsPlayerInCombat" that you could access & set anywhere.

Also you're setting on combat to false when we walk which doesn't really make sense. Better off just leaving the value to false all the time in the singleton for now and have proper logic to set that value added later instead.